website_docs/manual.md

```diff
@@ -7,9 +7,88 @@ aliases: [/docs/instrumentation/go/instrumentation, /docs/instrumentation/go/man
 
 Instrumentation is the process of adding observability code to your application. There are two general types of instrumentation - automatic, and manual - and you should be familiar with both in order to effectively instrument your software.
 
+## Initializing a Tracer
+
+To create spans, you'll need to initialize a tracer first. Some important things to consider:
+
+* Tracers should live at the module level and have a lifecycle congruent to an app's lifecycle
+* There should only be one tracer per module
+* Tracers need to be shut down when a program ends
+* You'll need an [exporter]({{< relref "exporting_data" >}}) to create a Tracer
+* You'll need to create Resources at trace initialization time
```

Comment:
Do you mean `TracerProvider` here where you say "tracer"?  Many of these statements are not true of `Tracer`s but are true of `TracerProvider`s.